concurrency-primer.tex

```diff
@@ -443,52 +464,54 @@ \subsection{Compare and swap}
 It allows us to conditionally exchange a value \emph{if} its previous value matches some expected one.
 In \clang{} and \cplusplus{}, \textsc{CAS} resembles the following,
 if it were executed atomically:
-\begin{cppcode}
-template <typename T>
-bool atomic<T>::compare_exchange_strong(
-    T& expected, T desired)
+\begin{ccode}
```

Comment:
This is borrowed from C11 7.17.4. Should I add reference as a footnote?

Reply:
> This is borrowed from C11 7.17.4. Should I add reference as a footnote?

Add the proper statement.

Reply:
Resolved in 4dc5699.